modeltrans/utils.py

```diff
@@ -25,4 +28,76 @@ def build_localized_fieldname(field_name, lang):
         # The 2-letter Indonesian language code is problematic with the
         # current naming scheme as Django foreign keys also add "id" suffix.
         lang = "ind"
-    return str("{}_{}".format(field_name, lang.replace("-", "_")))
+    return "{}_{}".format(field_name, lang.replace("-", "_"))
+
+
+def get_model_field(model, path):
+    """
+    Return the django model field for model in context, following relations.
+    """
+    if not hasattr(model, "_meta"):
+        return
+
+    field = None
+    for bit in path.split(LOOKUP_SEP):
+        try:
+            field = model._meta.get_field(bit)
+        except FieldDoesNotExist:
+            break
+
+        if hasattr(field, "remote_field"):
+            rel = getattr(field, "remote_field", None)
+            model = getattr(rel, "model", model)
+
+    return field
+
+
+def get_instance_field_value(instance, path):
```

Comment:
Deze is makkelijker te begrijpen. Ik denk dat ik bij get_model_field niet goed begrijp wat het perspectief is.
Hier is het een model instance, bij get model field is het de model class?

Reply:
Je analyse van het perspectief klopt!